src/containers/PageBuilder/SectionBuilder/SectionBuilder.module.css

```diff
@@ -6,12 +6,29 @@
   display: grid;
   justify-content: start;
   margin: 0 auto;
-  padding: 0 32px 64px;
+  padding: 0 32px;
   position: relative;
 
   @media (--viewportMedium) {
     justify-content: center;
   }
+
+  & + div {
+    padding: 64px 32px 0;
+  }
+}
+
+/* Reduce the white-space after the sectionTitle a bit more than in the Blocks */
+
+.sectionDetails {
+  & h1 + *,
+  & h2 + *,
+  & h3 + *,
+  & h4 + *,
+  & h5 + *,
+  & h6 + * {
+    margin-top: 12px;
+  }
```

Comment:
Don't declare .sectionDetails multiple times.
=> move these lines inside previous ".sectionDetails"
=> don't use wildcard/asterisk if you can avoid it.
=> keep the custom media query as the last thing inside class rules

```
.sectionDetails {
  max-width: var(--contentMaxWidth);
  display: grid;
  justify-content: start;
  margin: 0 auto;
  padding: 64px 32px 0 32px;
  position: relative;

  &:only-child {
    padding: 0 32px;
  }
  
  /* Reduce the white-space after the sectionTitle a bit more than in the Blocks */
  & h1 + *,
  & h2 + *,
  & h3 + *,
  & h4 + *,
  & h5 + *,
  & h6 + * {
    margin-top: 12px;
  }
  
  @media (--viewportMedium) {
    justify-content: center;
  }
}
```